tests/ha/priority_fencing_delay.pm

```diff
@@ -23,7 +23,7 @@ sub stonith_iptables {
         if (is_node(1)) {
             # Wait for the stonith match, then flush the rules for the next test
             script_run "until grep -qi offline <($crm_mon_cmd) ; do sleep 1; done", 60;
-            assert_script_run "iptables -F; iptables -X";
+            assert_script_run "iptables -F && iptables -X";
```

Comment:
I don't know why running the two commands together.
But the code change looks good to me.

Reply:
I wouldn't merge this without verification runs, as I don't remember exactly why a `;` was used or even if `assert_script_run` is appropriate here.

Both flags are used to delete rules and delete chains, so I'm not sure if it could happen that `iptables -F` fails (no rules to delete) and then `iptables -X` succeeds.

I did check on a system without defined rules, and it seems both commands return 0 in this scenario, but better confirm with a passing VR.

```
linux-mkji:~ # iptables -L
Chain INPUT (policy ACCEPT)
target     prot opt source               destination         

Chain FORWARD (policy ACCEPT)
target     prot opt source               destination         

Chain OUTPUT (policy ACCEPT)
target     prot opt source               destination         
linux-mkji:~ # iptables -F ; echo $?
0
linux-mkji:~ # iptables -X ; echo $?
0
```

Reply:
I added VR and please have a look

Reply:
Thanks!